apps/05Mastermind.js

```diff
@@ -29,12 +29,62 @@ function getRandomInt(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
-function generateHint() {
-  // your code here
+// Check to see if the guessed letters equal solution letters' location
+var correctLetterLocation = 0;
```

Comment:
You've defined `correctLetterLocations` and `correctLetters` and set them to `0` outside of the `generateSolution()` function. When you run the game or a series of tests, these 2 variables never get reset to zero after the user makes an incorrect guess. Also, make sure you have these 2 variables spelled correctly throughout your game.